docs/pages/older-versions.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Older Versions of Teleport Documentation
+description: Links to older versions of the Teleport Docs
+h1: GitHub links to older versions of the Teleport Docs
```

Comment:
```suggestion
```

I think we can use the title here instead of providing a separate H1.

Reply:
All pages need an H1 for SEO purposes @ptgott